File: aspnet-core/src/Selling_Website.Application.Contracts/Customers/Dto/CreateUpdateCustomerDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Selling_Website.Customers.Dto
{
    public class CreateUpdateCustomerDto : EntityDto<Guid?>
    {
        public string Name { get; set; }
        public DateTime? DOB { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
    }
}

File: angular/src/app/shared/service/customer/models/create-update-customer-dto.ts
import { EntityDto } from '@abp/ng.core';

export class CreateOrEditCustomerDto extends EntityDto<string>{
  name: string;
  dob: Date;
  address: string;
  email: string;
  phonenumber: string;
  message:string;
  subject:string;

  constructor(initial: Partial<CreateOrEditCustomerDto> = {}) {
    super(initial);
  }
}
